CONTRIBUTING.md

```diff
@@ -414,14 +414,11 @@ A PR is ready to merge when:
 * There is no `request changes` from the [codeowners](.github/CODEOWNERS)
 * If a change to the schema, at least one [example](examples/) is updated to illustrate change
 * All required status checks pass
-* The PR includes a [CHANGELOG](CHANGELOG.md) entry under `## UNRELEASED` following the form:
+* Has been tagged with any applicable [labels](#labels)
 
-```markdown
+### Labels
 
-* {Brief description of change}
-  ([#{PR Number}]({https://github.com/open-telemetry/opentelemetry-configuration/pull/{PR Number}))
```

Comment:
Deleting this because we haven't been following this. Instead, we've been summarizing the changes as part of the release process.